services/prefect/flows/scheduled_nyt_and_parquet_updater.py

```diff
@@ -1,17 +1,52 @@
-from prefect import Flow
+import requests
+from datetime import timedelta
+from prefect import Flow, task
+from prefect.engine.state import Failed
 from prefect.schedules import CronSchedule
+from prefect.tasks.secrets import PrefectSecret
+from prefect.utilities.notifications import slack_notifier
 from prefect.tasks.prefect import create_flow_run, wait_for_flow_run
 
 
+
+@task(max_retries=3, retry_delay=timedelta(minutes=1))
+def make_request(github_token):
+    dispatch_url = (
+        "https://api.github.com/repos/covid-projections/covid-data-model/"
+        "actions/workflows/update_repo_datasets.yml/dispatches"
+    )
+    response = requests.post(
+        url=dispatch_url,
+        headers={
+            "Authorization": f"token {github_token}",
+            "Accept": "application/vnd.github.v3+json",
+        },
+        data='{"ref": "main"}',
+    )
+    response.raise_for_status()  # raise status in case of failure
+
+
+def trigger_data_model_flow():
```

Comment:
Nit: Do we have a naming convention for naming flows?  Seems a bit confusing that `init_scheduled_nyt_updater_flow()` is prefixed with `init_` but this one isn't... But maybe there's a difference I'm not understanding.

Reply:
We don't, yeah, we should make this more uniform. They both do the same thing--I'll update this one to match `init_scheduled_nyt_updater_flow` when I get back to this